src/test/scala/cromwell/engine/db/slick/SlickDataAccessSpec.scala

```diff
@@ -137,13 +135,13 @@ class SlickDataAccessSpec extends FlatSpec with Matchers with ScalaFutures {
       val task = new Task("taskName", Nil, Nil, Nil, null, BackendType.LOCAL)
       val callFqn = "fully.qualified.name"
       val call = new Call(None, callFqn, task, Set.empty[FullyQualifiedName], Map.empty, None)
-      val backendInfo = new LocalCallBackendInfo(ExecutionStatus.Running, Option(123), Option(456))
+      val backendInfo = new LocalCallBackendInfo(CallStatus(ExecutionStatus.Running, None), Option(456))
 
       (for {
         _ <- dataAccess.createWorkflow(workflowInfo, Nil, Seq(call), localBackend)
         _ <- dataAccess.updateWorkflowState(workflowId, WorkflowRunning)
         _ <- dataAccess.getExecutionStatus(workflowId, ExecutionDatabaseKey(callFqn, None)) map { status =>
-          status.get shouldBe ExecutionStatus.NotStarted
+          status.get.executionStatus shouldBe ExecutionStatus.NotStarted
```

Comment:
`CallStatus.rc` is not also checked. Intellij shows there are 8 checks of `.executionStatus`, but only 3 corresponding checks of `.rc` in this test.